UnityProject/Assets/Scripts/MapSaver/MapSaver.cs

```diff
@@ -0,0 +1,176 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Text;
+using TileManagement;
+using UnityEngine;
+using System.Linq;
+
+public static class MapSaver
+{
+	public struct TileMapData
+	{
+		public List<string> CommonColours;
+		public List<string> CommonLayerTiles;
+		public List<string> CommonMatrix4x4;
+		public string Data;
+	}
+
+	public static TileMapData SaveTileMap(MetaTileMap MetaTileMap)
+	{
+		//# Matrix4x4
+		//§ TileID
+		//◉ Colour
+
+		//☰ Layer
+		//@ location
```

Comment:
ENUMS PLEASE

Reply:
uhm, enums are named numbers so it wouldn't work to use instead of the characters Bod is using to serliaze the map. A class to define the constants would work.

```cs
public static class MapSerializationConstants
{
    public static string Matrix = "#";
    publis static string TileID = "§";
    ...
}
```

Reply:
oh yeah, characters are used for improving the readability

Reply:
use constants so it is easier to swap them in the future if we need